fiftyone/core/state.py

```diff
@@ -113,7 +117,7 @@ def serialize(self, reflective=True):
                 ]
 
                 view = self.view if self.view is not None else self.dataset
-                if view.media_type == fom.GROUP:
+                if view.group_slice:
```

Comment:
This was breaking the two tab issue I was talking about. I had to bring it back and modify to view.group_slice for the listing tab to get the updated groupSlice through the session

Reply:
Ok, adding this will break another edge case. Let's discuss offline tomorrow

Reply:
I reverted this for another edge case, but let's follow up offline so we can fix